virtcontainers/network.go

```diff
@@ -1277,6 +1432,28 @@ func generateInterfacesAndRoutes(networkNS NetworkNamespace) ([]*grpc.Interface,
 	return ifaces, routes, nil
 }
 
+func createNetworkInterfacePair(idx int, ifName string, interworkingModel NetInterworkingModel) NetworkInterfacePair {
```

Comment:
Is the index in the codebase correctly updated? I am asking cause otherwise the hardware address will end up being not unique.

Reply:
Yes. It is passed correctly.